conversions/hexadecimal_to_octal2.c

```diff
@@ -112,8 +112,8 @@ int main()
 
     const char *octal = hex_to_oct(hex);
 
-    printf("Hexadecimal number = %s\n", hex);
-    printf("Octal number = %s\n", octal);
+    printf("Hexadecimal number: %s", hex);
+    printf("Octal number: %s\n", octal);
```

Comment:
Improve output formatting.
Change seperator make consistency with the rest code snippets.